app/ide-desktop/lib/client/src/file-associations.ts

```diff
@@ -0,0 +1,65 @@
+import pathModule from 'node:path'
```

Comment:
no module docs - linter should catch it. I've written about it in my previous code review - could you please check linter config and check why it did not report here?

Reply:
@wdanilo  I failed to find where the linter is invoked. I tried running `./run lint`, and it found no issues, though I don't see it linted anything. How is the TS linting supposed to work now?